libminifi/include/controllers/SSLContextService.h

```diff
@@ -72,20 +72,20 @@ class SSLContext {
  * Justification: Abstracts SSL support out of processors into a
  * configurable controller service.
  */
-class SSLContextServiceImpl : public core::controller::ControllerServiceImpl, public SSLContextService {
+class SSLContextService : public core::controller::ControllerServiceImpl, public SSLContextServiceInterface {
  public:
-  explicit SSLContextServiceImpl(std::string_view name, const utils::Identifier &uuid = {})
+  explicit SSLContextService(std::string_view name, const utils::Identifier &uuid = {})
       : ControllerServiceImpl(name, uuid),
         initialized_(false),
-        logger_(core::logging::LoggerFactory<SSLContextService>::getLogger(uuid_)) {
+        logger_(core::logging::LoggerFactory<SSLContextServiceInterface>::getLogger(uuid_)) {
```

Comment:
we should keep the class name in the log as "SSLContextService" (also in line 86)